src/App.tsx

```diff
@@ -1,14 +1,67 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { actions as todosActions } from './features/todos';
+import { getTodos } from './api';
 
 export const App: React.FC = () => {
+  const [isLoader, setIsLoader] = useState<boolean>(false);
+
+  const dispatch = useAppDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const { status, query } = useAppSelector(state => state.filter);
+
+  const lowerQuery = query.toLowerCase();
+
+  const getData = async () => {
+    try {
+      setIsLoader(true);
+      let loadedData = await getTodos();
+
+      switch (status) {
+        case 'all':
```

Comment:
It is better to consider `all` case as default in `switch` instead of making it explicit.